### DISCUSSION TITLE
Feature request - automatic driver install

### DISCUSSION THREAD

**Discussion by turdferguson68** [RaspAP/raspap-webgui#1032] [Ideas]
Found a script that installs drivers for the most popular driver chips:

http://downloads.fars-robotics.net/wifi-drivers

subject script is /install-wifi

I got it running with the following:

wget http://downloads.fars-robotics.net/wifi-drivers/install-wifi -O /usr/bin/install-wifi
sudo chmod +x /usr/bin/install-wifi
sudo install-wifi

I know dongle drivers have been a problem in the past. Maybe this could be added to the script?

**Comment by billz** [comment]
See https://github.com/RaspAP/raspap-tools 😉
